src/ApplicationManager.cpp

```diff
@@ -53,28 +53,28 @@ void ApplicationManager::start() {
 
     plays.emplace_back(std::make_unique<rtt::ai::stp::play::AttackingPass>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::Attack>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::Halt>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::DefendShot>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::DefendPass>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::DefensiveStopFormation>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::AggressiveStopFormation>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::BallPlacementUs>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::BallPlacementThem>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::TimeOut>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyThemPrepare>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyUsPrepare>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyThem>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyUs>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffUsPrepare>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffThemPrepare>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::FreeKickThem>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffUs>());
-    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffThem>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::Attack>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::Halt>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::DefendShot>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::DefendPass>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::DefensiveStopFormation>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::AggressiveStopFormation>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::BallPlacementUs>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::BallPlacementThem>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::TimeOut>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyThemPrepare>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyUsPrepare>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyThem>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::PenaltyUs>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffUsPrepare>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffThemPrepare>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::FreeKickThem>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffUs>());
+//    plays.emplace_back(std::make_unique<rtt::ai::stp::play::KickOffThem>());
```

Comment:
This is done due to it being a temp disable of most of these plays. They will be re-enabled when the basis of a play is done.

Reply:
Well yeah, but you guys wanted to see a PR of where we currently are right? Otherwise the PR would get way too big. We're not done with the end goal yet, so we still have things like this commented out for testing

Reply:
"Commented out for testing"

Fine; but then just comment it back in on a branch you continue working with again. We don't want commented code committed to development. Atleast, if you didn't break anything.

Reply:
I actually dont think we'll use this branch for developing/testing anymore, so ill just uncomment everything for this PR so we can (hopefully) merge it

Reply:
yeah that's good